### PR TITLE
Enable parallel compile & tests on gps-raspbian.

### PR DESCRIPTION
**Pull request by gpshead**

### CODE REVIEW

**Comment by gpshead** [comment]
Looks like buildbot doesn't autodetect cpu & ram, manually specify the parallelization here. @zware

File: master/custom/workers.py [hunk 1]
```diff
@@ -192,6 +192,8 @@ def get_workers(settings):
             name="gps-raspbian",
             tags=['linux', 'unix', 'raspbian', 'debian', 'armv6', 'armv7l',
                   'aarch32', 'arm'],
+            parallel_builders=5,
+            parallel_tests=7,
```
Comment (zware) [inline master/custom/workers.py:196]: This could allow up to 35 tests running at a time if we have 5 active branches all building; is that the intention?

Comment (gpshead) [reply master/custom/workers.py:196]: hah, nope.  I just want it `make all -j 5` and `python -m test.regrtest -j 7`.

i'm used to seeing buildbot serialize access to the bot.  if 3.x is building, 3.8 will be queued waiting until that one finishes, etc.  But appropriate build parallelism gets used within the individual builds.

The logs show serial make and -j 2 being passed to the tests right now.  that is a waste.

It is a four core machine, it needs 4+1 tasks.

Comment (gpshead) [reply master/custom/workers.py:196]: It looks like `custom/factories.py` needs a _new class defined_ *!!* and I have to update the `custom/builders.py` file to reference that instead?

sigh.  buildbot makes this so rediculious.

Comment (gpshead) [reply master/custom/workers.py:196]: master.cfg makes it look like setting _only_ `parallel_tests=6` will do what I want?  that one appears to get turned into a -j flag via a few levels of indirection.  Which then winds up passed as parallel="-j6" to the UnixBuilder factory specified in builders.py presumably winding up passed to setup where it finds it way to the make all and make buildbot test command line.

Comment (zware) [reply master/custom/workers.py]: Yes, things are a bit convoluted :(.  You did get it straightened out, though; sorry for not just laying that out in my previous comment!

**Pull request merged**